### PR TITLE
Feature/claudia 5389 refactoring

### PR DESCRIPTION
**Pull request by flopezag**
### REVIEWERS
@chemaper, @jframos  

### DESCRIPTION
- Solution of Tech. Debt related to print statement and too high complexity methods.
- Resolution of coding style problems.
- Refactorization of code.
- Migration from lettuce to behave.
- Update README files.

### TESTING
- Executed locally unit tests and acceptance tests.
- Passing the execution in Jenkins.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [report]
 omit =
     tests/*
-    settings/*
+    conf/*
```
Comment (jframos) [inline .coveragerc]: Do we need include configuration files as part of coverage measures?

Comment (flopezag) [inline .coveragerc]: we are not including it...

Nevertheless, I need previously resolve the merge.

Comment (jframos) [inline .coveragerc]: opsss, sorry.... I didn't see the _omit_

File: phase2_stopvms.py [hunk 2]
```diff
@@ -22,19 +22,17 @@
...
 import datetime
 
-from user_resources import UserResources
-from settings.settings import TRUSTEE
-import settings.settings
+from skuld.user_resources import UserResources
+import conf.settings
 import utils.log
 
+author = 'chema'
```
Comment (jframos) [inline phase2_stopvms.py]: I prefer to use native python object syntax for this type of variables, like this: \__author__

File: tests/acceptance/features/steps/keystone_token.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+from tests.acceptance.commons.configuration import KEYSTONE_URL, TOKEN_LENGTH
+
+__author__ = 'fla'
+
+
+MULTIPLE_CHOICE = 300
+
+
+@given(u'a valid tenantName, username and password')
+def step_impl(context):
```
Comment (jframos) [inline tests/acceptance/features/steps/keystone_token.py]: I can see a lot of methods named with the same name _step_impl_. I prefer using different ones for each function.

File: tests/acceptance/resources/logging.conf [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+qualname=qautils
+propagate=0
+
+[logger_requests]
+level=ERROR
+handlers=consoleHandler,fileHandler
+qualname=requests.packages.urllib3.connectionpool
+propagate=0
+
+[logger_paramiko]
```
Comment (jframos) [inline tests/acceptance/resources/logging.conf]: I think paramiko lib is not used by these acceptance tests.

**Comment by jframos** [comment]
The branch is in conflict with develop branch.

**Comment by jframos** [comment]
Perhaps we can move all _phaseX_*.py_ scripts into a new folder in _scripts/_. This way we will have organized all scripts taking part of this component, like _unacceptedTermsAndConditions_. New scripts for user management will be included there, not only for trial user management. What do you think?

**Comment by flopezag** [comment]
The problem is that the phaseX.py scripts are the main component of this repo, the idea should be integrate all of them in a single file or create a master python script which call all of them depending of the CLI parameters.

**Comment by chemaper** [comment]
I think that analyse_report_data.py and purge_images.py should be moved from skuld/ to scripts

**Comment by chemaper** [comment]
The scrips create_resources.py and list_resources.py are referenced in the README.rst with their old paths (in the section about testing, tools). The README.rst should be updated, either with the right paths or removing the mention to the files.

File: phase2b_detectimagesinuse.py [hunk 5]
```diff
@@ -22,12 +22,13 @@
 # For those usages not covered by the Apache version 2.0 License please
 # contact with opensource@tid.es
 #
-author = 'chema'
-
-from queries import Queries
 import cPickle as pickle
+
+from skuld.queries import Queries
 import utils
```
Comment (chemaper) [inline phase2b_detectimagesinuse.py]: This line should be import utils.log. Otherwise there is an error.

**Comment by chemaper** [comment]
LGTM

**Comment by jframos** [comment]
"LGTM"

**Pull request merged**